src/pip/_internal/commands/configuration.py

```diff
@@ -226,10 +226,8 @@ def _save_configuration(self):
         try:
             self.configuration.save()
         except Exception:
-            logger.error(
-                "Unable to save configuration. Please report this as a bug.",
-                exc_info=1
-            )
+            logger.exception("Unable to save configuration. "
+                             "Please report this as a bug.")
```

Comment:
Nit: Would it be better to follow the same format of keeping the entire log message in a new line as before, instead of breaking it in two lines?

Reply:
I don't have a strong opinion on this, but I don't really like the dedented closing parenthesis, especially when the enclosed content is a single line.  IMHO the style is to facilitate adding more arguments in the same call w/o diff noise later, but this is not the case and dedented for closing of block doesn't look very Pythonic to me.  That being said, I'm fine if we have a consensus (e.g. black, which is yet to be applied for pip ATM) on what you recommended.

Reply:
Well, consistency within the module is important, and this module definitely follows this style. That on it's own is reason enough to maintain the style as it was originally. 

Generally, I think it's a good idea to avoid making changes that could initiate code-style discussions in PRs. A good way to do that is to follow the style from the original code as much as possible/reasonable.

Additionally, given that the plan is to adopt black eventually, I think it'd be a good thing to minimise the diff wherever reasonable.